test/unit/consultation_test.rb

```diff
@@ -67,8 +67,8 @@ class ConsultationTest < ActiveSupport::TestCase
   test "should build a draft copy of the existing consultation with inapplicable nations" do
     published_consultation = create(:published_consultation, nation_inapplicabilities: [
       create(:nation_inapplicability, nation_id: Nation.wales.id, alternative_url: "http://wales.gov.uk"),
-      create(:nation_inapplicability, nation_id: Nation.scotland.id, alternative_url: "http://scot.gov.uk")]
-    )
+      create(:nation_inapplicability, nation_id: Nation.scotland.id, alternative_url: "http://scot.gov.uk")
+])
```

Comment:
It would be better to break up and indent this `])` so the `]` lines up with the `nation_inapplicabilities: [` that opened it and the `)` lines up with the `create(` that opened it (as they're on the same line we might want to break them up too).